dotnet-etcd/dotnet-etcd.csproj

```diff
@@ -46,6 +44,7 @@ Advanced uses take advantage of the consistency guarantees to implement database
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="DnsClient" Version="1.2.0" />
```

Comment:
Anything specific this library can do that `Dns` in `System.Net` can't ?

Reply:
System.Net.Dns resolves hostnames, doesn't do special records from what I can tell. If you know otherwise, please point it out.

Reply:
I see that you are trying to retrieve srv records. 
Something like this would help: 
`https://stackoverflow.com/a/17913119`

I don't want to include an external library if it is possible to do it out of the box.

Reply:
I don't think that is a good path to take as it makes the library dependent on Windows native libraries while the DnsClient is a native .NET library (as in runs on all platforms -- Windows, Linux, Mac, etc.).

Reply:
Fair Enough. Until we have an out of the box implementation, we will have to live with this.